dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/oracle/V5.7_2017.04.11__DS-3563_Index_metadatavalue_resource_type_id_column.sql

```diff
@@ -19,5 +19,5 @@ begin
   exception
   when index_not_exists then null;
 end;
-
+/
```

Comment:
Is this change required to this old Migration script?  Modifying a previous migration script, even by a single character will change the checksum of that script.  That means that Flyway will no longer be able to validate this migration on databases that previously ran it.  So, with this one character change, anyone running 5.7 or 5.8 will be *required* to run a `dspace database repair` in order to upgrade successfully.  See https://wiki.duraspace.org/display/DSDOC6x/Database+Utilities